ConcealedCard[] concealed_card_list
ExposedCard[] exposed_card_list
Position[] no_card_list